### ISSUE TITLE
Plugin menu buttons are in different style

### ISSUE THREAD

**Issue by Neui** [getting-things-gnome/gtg#383]
It looks like menu buttons (in the main menu) added via plugins have different styling, such as being centred and has different "internal structure":

![image](https://user-images.githubusercontent.com/1196130/85237824-e9f49300-b429-11ea-8e6d-ee7c5cc319d2.png)

I couldn't really find any way to easily add "as expected" without trying to recreate the "internal structure" (MenuButton with a Box with a Label in it).
All I've found is either people using Builder files (not useful in our case), or using GMenu (not sure if/how you could extend using plugins).
An idea would be (if it is possible in GTK) to copy an existing button (such as the "hide sidebar" button), and change it's label (after trying to find it).

**Comment by diegogangl** [comment]
We already have an API method to add these, so we could just improve it to replicate the internal structure of the others items.

**Comment by Neui** [comment]
True, however my main worry is that since I couldn't really find where that is document, so that the final "internal structure" could change on the next GTK (3) version (although admittedly they are rather working on GTK 4, so it is possible that GTK 3 would become "stable" in that regard.)

**Comment by diegogangl** [comment]
There's no future GTK3 versions AFAIK, only bug fixes. And if it changes we just have to fix that method, I don't expect it to happen often
